### DISCUSSION TITLE
how to make page private

### DISCUSSION THREAD

**Discussion by LEESHOHYUN** [mmistakes/minimal-mistakes#3108] [Q&A] [answered]
hello. I fork your minimal-mistakes to make my own blog
and I want to make that blog private. Does anyone knows how to make it privately?

**Comment by iBug** [comment, marked as answer]
Delete your fork and re-create the repository.

Also the recommended way to create your website is with [the starter repo](https://github.com/mmistakes/mm-github-pages-starter/generate).

**Reply by LEESHOHYUN** [reply]
so I have to make new repo, and copy files of minimal-mistakes, you mean? am I understand right? and also, should I have to include all branches or not?

**Reply by LEESHOHYUN** [reply]
I made a repo with your link, but I don't know how to make it private. I'm sorry but may I ask more help??

**Reply by iBug** [reply]
Most questions are easily answered with Google. This is the first result that turned up when searching for "github turn repo to private":

<https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/setting-repository-visibility>

**Reply by LEESHOHYUN** [reply]
oh... I want page private. not repositories private.

**Reply by iBug** [reply]
No you can't. GitHub Pages website is always public regardless of the visibility of the repository itself.

**Reply by LEESHOHYUN** [reply]
okay I understand, Thx for the reply!
